site/source/doc/manual/configuration.md

```diff
@@ -138,12 +178,8 @@ mode.configure 'c', {
   indent: 4
 }
 
--- Hook up a signal handler to set it to three for this weird project
-that_project_root = File '/home/nino/code/that_project'
-
-signal.connect 'file-opened', (args) ->
-  if args.file\is_below that_project_root
-    args.buffer.config.indent = 3
+-- Set it to three for this weird project
+config.for_file('/home/nino/code/some_project').indent = 3
```

Comment:
I actually use 3 space indent for everything not part of others' projects - It's the perfect indent!
But "weird" sounds cool :-)